src/api/app/views/webui2/webui/package/_show_failures.html.haml

```diff
@@ -0,0 +1,6 @@
+%li
+  %i.fas.fa-times-circle.text-danger{ title: 'Errors' }
+  = failures
+  = link_to "error#{failures == 1 ? '' : 's'}", project_monitor_path(project: project, pkgname: package_name, succeeded: 0,
```

Comment:
Nice use case for [pluralize](https://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#method-i-pluralize)

Reply:
@dmarcoux, good catch